strongbox.go

```diff
@@ -3,7 +3,9 @@ package main
 import (
 	"bytes"
 	"compress/gzip"
+	"crypto"
 	"crypto/rand"
+	_ "crypto/sha256"
```

Comment:
won't work without it because it's init function registers the hash for use (actually tests still pass, presumably because something in the testing dependency tree pulls it in)